CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Changed
+- Docker - Use `alpine` as base node image.
```

Comment:
```suggestion
- Command Generate - Use `alpine` as base node Docker image for generated projects.
```

Reply:
@thatguydan Could you commit this suggestion? (I "do not have permission to push to this repository", Github says when I click on "commit suggestion", probably because it's on a fork).